src/App.jsx

```diff
@@ -1,51 +1,105 @@
-            <Routes>
-              {/* Add routes as needed; route names subject to change */}
-              <Route exact path="/" element={<HomePage />} />
-              <Route exact path="/account" element={<AccountPage />} />
-              <Route exact path="/create-log" element={<MonitorLogPage />} />
-              <Route exact path="/sections" element={<SectionPage />} />
-              <Route
-                exact
-                path="/endangered"
-                element={<EndangeredSpeciesPopup adultName="Snowy Plovers" />}
-              />
-              <Route exact path="/species" element={<Species />} />
-              {/* Admin only routes (TO DO, make admin only) */}
-              <Route exact path="/people" element={<PeoplePage />} />
-              <Route exact path="/contacts" />
-              <Route exact path="/map" />
-              <Route exact path="/logs" element={<AdminPage />} />
-              <Route exact path="/common-table-example" element={<CommonTableExample />} />
-            </Routes>
+      <CookiesProvider>
+        <Router>
+          <Box className="page-container">
+            <Box className="content-wrap">
+              <Navbar isAdmin />
+              <Routes>
+                {/* Add routes as needed; route names subject to change */}
+                <Route path="/invite-user/:inviteID" element={<InviteLandingPage />} />
+                <Route exact path="/invite-modal" element={<AdminInviteModal />} />
+                <Route exact path="/" element={<HomePage />} />
+                <Route exact path="/account" element={<AccountPage />} />
+                <Route exact path="/create-log" element={<MonitorLogPage />} />
+                <Route exact path="/sections" element={<SectionPage />} />
+                <Route
+                  exact
+                  path="/endangered"
+                  element={<EndangeredSpeciesPopup adultName="Snowy Plovers" />}
+                />
+                <Route exact path="/species" element={<Species />} />
+                {/* Admin only routes (TO DO, make admin only) */}
+                <Route exact path="/people" element={<PeoplePage />} />
+                <Route exact path="/contacts" />
+                <Route exact path="/map" />
+                <Route exact path="/logs" element={<AdminPage />} />
+                <Route exact path="/common-table-example" element={<CommonTableExample />} />
+
+                {/* NEW AUTH ROUTES */}
+                <Route path="/login" element={<Login />} />
+                <Route
+                  path="/admin"
+                  element={
+                    <ProtectedRoute
+                      Component={Logout}
+                      redirectPath="/logout"
+                      roles={[SUPER_ADMIN_ROLE, ADMIN_ROLE]}
+                    />
+                  }
+                />
+                <Route path="/emailAction" element={<EmailAction redirectPath="/" />} />
```

Comment:
Inconsistency: when we send an email to the user, the path is `/email-action`, so nothing shows up because it's on `/emailAction`. Please fix this to make it work and make it consistent